src/SplitPackages/ProjectJsonFileBuilder.cs

```diff
@@ -131,7 +131,7 @@ public void Execute()
 
         private IDictionary<string, JObject> CreateRuntimesDictionary()
         {
-            return new[] { Runtime.Win7x64, Runtime.Win7x86 }.ToDictionary(r => r.Name, r => new JObject());
+            return Runtime.AllRuntimes.ToDictionary(r => r.Name, r => new JObject());
```

Comment:
Where does `AllRuntimes` come from? I don't see it in `src/SplitPackages/Runtime.cs`, is it an extension from somewhere?